main.cpp

```diff
@@ -0,0 +1,59 @@
+#include <iostream>
+#include <vector>
+#include "Functions.h"
+#include "Animal.h"
+#include "Functions.cpp"
+
+using namespace std;
+
+
+int main()
+{
+    vector<Animal> animals;
+    int choice;
+
+    do
+    {
+        cout << "Ferma contine " << animals.size() << " animale." << endl;
+        choice = showMenu();
+        switch(choice)
```

Comment:
aici poti sa dai direct `switch(showMenu())` daca faci cum am zis mai jos cu while true si return in case, si asa nu mai trebuie sa salvezi ca variabila si `choice`

o chestie de personal preference as zice, dar eu asa as face